ext/random/tests/02_engine/all_serialize.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+Random: Engine: serialize
+--FILE--
+<?php
+
+final class User64 implements \Random\Engine
+{
+    private int $count = 0;
+
+    public function generate(): string
+    {
+        return \pack('P*', ++$this->count);
+    }
+}
+
+final class User32 implements \Random\Engine
+{
+    private int $count = 0;
+
+    public function generate(): string
+    {
+        return \pack('V', ++$this->count);
+    }
+}
+
+$engines = [];
+$engines[] = new \Random\Engine\Mt19937(1234);
+$engines[] = new \Random\Engine\PcgOneseq128XslRr64(1234);
+if (\PHP_INT_SIZE >= 8) {
+    $engines[] = new \User64();
+}
```

Comment:
```suggestion
$engines[] = new \User64();
```
This test should even work on a 32 bit system.

Reply:
Pack function call fails:

```
001+ Fatal error: Uncaught ValueError: 64-bit format codes are not available for 32-bit versions of PHP in /home/vsts/work/1/s/ext/random/tests/02_engine/all_serialize.php:9
001- success
002+ Stack trace:
003+ #0 /home/vsts/work/1/s/ext/random/tests/02_engine/all_serialize.php(9): pack('P*', 1)
004+ #1 /home/vsts/work/1/s/ext/random/tests/02_engine/all_serialize.php(31): User64->generate()
005+ #2 {main}
006+   thrown in /home/vsts/work/1/s/ext/random/tests/02_engine/all_serialize.php on line 9
```

Reply:
Oh, I see. Disregard the remark then.